src/index.css

```diff
@@ -2,4 +2,5 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  background-color: white;
```

Comment:
Were the instructors trying to trick us by displaying black text on a black background??? Why so lit tho